pkg/apis/messaging/v1/subscription_validation.go

```diff
@@ -62,6 +62,7 @@ func (ss *SubscriptionSpec) Validate(ctx context.Context) *apis.FieldError {
 		}
 	}
 
+	missingReply := isDestinationNilOrEmpty(ss.Reply)
 	if !missingReply {
```

Comment:
Why not collapse `missingReply` into the if statement, since it's only used once?

Reply:
yup.. done